tensorflow/python/keras/layers/normalization.py

```diff
@@ -945,6 +950,25 @@ def __init__(self,
     self.gamma_constraint = constraints.get(gamma_constraint)
 
     self.supports_masking = True
+    self.fused = fused
+
+  def _raise_if_fused_cannot_be_used(self, ndims):
+    """Raises a ValueError if fused implementation cannot be used.
+
+    Check if the axis is contiguous and can be collapsed into the last axis.
+    """
+    axis = sorted(self.axis)
+    err = False
+    if axis[-1] - axis[0] != len(axis) - 1:
```

Comment:
I would swap these two if-statements, since what's currently the first if-statement doesn't make much sense unless you know about the second. And, add a comment this tests if the axis is contiguous, since you know axis does not have duplicate values.

Or, just use an explicit for-loop, to assert each axis element is the correct value. Performance isn't a concern here.